### ISSUE TITLE
strange training log

### ISSUE THREAD

**Issue by ShaobinChen-AH** [TXH-mercury/VALOR#30]
hello, I am finetuning the valor model on my dataset with settings and hyperparameters you mentioned. However, I get many training logs like this 
" WARNING - tensorboardX.x2num -   NaN or Inf found in input tensor.
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 512.0"

did you get these? how to solve it?
